.github/workflows/publish-page.yml

```
@@ -20,6 +26,9 @@ concurrency: ci-${{ github.ref }}
 
 jobs:
   publish:
+    # Do not run this on forks:
+    if: github.repository == 'grafana/tanka'
```

this is what you have to do to detect a fork:

```suggestion
    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'grafana/tanka'
```

`github.repository` is which repo the event is running _for_ - for example it's a pull request _for_ `grafana/tanka` so these will still run 😞

---

Argh, thank you 🙂